cmd/helm/get.go

```diff
@@ -47,6 +48,9 @@ func newGetCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 		Long:  getHelp,
 		Args:  require.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			kc := kube.New(kubeConfig())
+			kc.IsReachable()
```

Comment:
Can you please move this to the [`action` package](https://github.com/helm/helm/tree/dev-v3/pkg/action)? In this code flow you'll find it inside `client.Run()`. That way if someone imports the action package to leverage this functionality they will have checks for for connectivity.